definitions/divorce/json/FixedLists/FixedLists.json

```diff
@@ -1266,18 +1266,6 @@
     "ListElement": "Cheque",
     "DisplayOrder": 4
   },
-  {
-    "LiveFrom": "23/07/2020",
-    "ID": "d8documentsReceivedEnum",
```

Comment:
Wait, was that released already?

Reply:
@Dizrupto Good question. When was our last release, maybe safer to leave where it was. Doesnt hurt just though to make the feature more obvious.